modules/wpcom-block-editor/class-jetpack-wpcom-block-editor.php

```diff
@@ -279,12 +280,23 @@ public function enqueue_block_editor_assets() {
 			true
 		);
 
+		// phpcs:ignore WordPress.Security.NonceVerification
+		$editor_deprecated               = isset( $_GET['editor/after-deprecation'] );
+		$classic_editor_plugin_available = is_plugin_inactive( 'classic-editor/classic-editor.php' );
+		$switch_visible                  = ( ! $editor_deprecated || jetpack_is_atomic_site() ) && $this->is_iframed_block_editor() && $classic_editor_plugin_available;
```

Comment:
I think we can remove the `jetpack_is_atomic_site` check. This should work on Jetpack sites as long as they have the classic editor installed. If they don't we won't show the link.

I think we should do that to remove the differences between an atomic site and a Jetpack site.